bang/magicollections.py

```diff
@@ -697,10 +697,10 @@ def to_fields(self, item, extra_fields=None, exclude_fields=None, order=None, *a
                     } for cameo in item.cached_cameos]
                 }))
             # Add live2d viewer
-            if hasattr(item, "associated_costume"):
+            if hasattr(item, 'associated_costume'):
```

Comment:
You're right. Looks like it's only always present when it's a direct foreign key and not a reverse lookup as I assumed.